packages/graphql-yoga/__tests__/health-checks.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { createYoga } from 'graphql-yoga'
+
+describe('health checks', () => {
+  it('return 200 status code for health check endpoint', async () => {
+    const yoga = createYoga({
+      logging: false,
+    })
+    const result = await yoga.fetch('http://yoga/health', {
+      method: 'GET',
+    })
+    expect(result.status).toBe(200)
+    expect(await result.json()).toEqual({
+      message: 'alive',
+    })
+  })
+  // TODO: this currently requires a "true" HTTP request :thinking:
+  it('return 200 status code for readiness check endpoint', async () => {
+    const yoga = createYoga({
+      logging: false,
+    })
+    const result = await yoga.fetch('http://yoga/readiness', {
+      method: 'GET',
+    })
+    expect(result.status).toBe(200)
+    expect(await result.json()).toMatchInlineSnapshot()
+  })
```

Comment:
This test can currently only pass as an integration test as it is doing a real HTTP request. I raised this concern on Slack as I think that we should not do a real HTTP request here. (See https://tortilla-hq.slack.com/archives/C02HP79RCLR/p1660116166112359)

Reply:
https://tortilla-hq.slack.com/archives/C02HP79RCLR/p1660116166112359

Reply:
moved to integration tests for now.